core/assets/bundles/bundle_cs.properties

```diff
@@ -616,25 +616,25 @@ filter.option.floor2 = Druhotný  povrch
 filter.option.threshold2 = Druhotný práh
 filter.option.radius = Poloměr
 filter.option.percentile = Percentil
-filter.option.code = Code
-filter.option.loop = Loop
-locales.info = Here, you can add locale bundles for specific languages to your map. In locale bundles, each property has a name and a value. These properties can be used by world processors and objectives using their names. They support text formatting (replacing placeholders with actual values).\n\n[cyan]Example property:\n[]name: [accent]timer[]\nvalue: [accent]Example timer, time left: {0}[]\n\n[cyan]Usage:\n[]Set it as objective's text: [accent]@timer\n\n[]Print it in a world processor:\n[accent]localeprint "timer"\nformat time\n[gray](where time is a separately calculated variable)
-locales.deletelocale = Are you sure you want to delete this locale bundle?
-locales.applytoall = Apply Changes To All Locales
-locales.addtoother = Add To Other Locales
-locales.rollback = Rollback to last applied
-locales.filter = Property filter
-locales.searchname = Search name...
-locales.searchvalue = Search value...
-locales.searchlocale = Search locale...
-locales.byname = By name
-locales.byvalue = By value
-locales.showcorrect = Show properties that are present in all locales and have unique values everywhere
-locales.showmissing = Show properties that are missing in some locales
-locales.showsame = Show properties that have same values in different locales
-locales.viewproperty = View in all locales
-locales.viewing = Viewing property "{0}"
-locales.addicon = Add Icon
+filter.option.code = Kód
+filter.option.loop = Smyčka
+locales.info = Zde můžeš přidat lokální balíčky pro konkrétní jazyky tvé mapy. V lokálních balíčcích má každá vlastnost jméno a hodnotu. Tyto vlastnosti mohou být použity světovými procesory a úkoly pomocí jejich jmen. Podporují textové formátování (nahrazení dočasného obsahu skutečnými hodnotami).\n\n[cyan]Příkladová vlastnost:\n[]jméno: [accent]časovač[]\nhodnota: [accent]Příkladový časovač, zbývající čas: {0}[]\n\n[cyan]Použití:\n[]Nastavení jako text úkolu: [accent]@timer\n\n[]Vypsání ve světovém procesoru:\n[accent]localeprint "timer"\nformat time\n[gray](kde čas je odděleně vypočítaná proměnná)
+locales.deletelocale = Jsi si jistý, že chceš smazat tento lokální balíček?
+locales.applytoall = Aplikovat změny ve všech balíčcích
+locales.addtoother = Přidat do ostatních balíčků
```

Comment:
Do "Locales" refer to the local bundles ? That's how i translated it, but just wanted to make sure.